libs/blocks/accordion/accordion.js

```diff
@@ -64,4 +65,6 @@ export default function init(el) {
   el.classList.add('con-block', maxWidthClass || 'max-width-10-desktop');
   accordion.classList.add('foreground');
   el.append(accordion);
+  decorateBlockAnalytics(el);
```

Comment:
Small nit: I think the analytics decoration could be moved before the actual elements are added to the DOM.

Reply:
True, this is updated. Thanks